emission/analysis/modelling/tour_model/representatives.py

```diff
@@ -142,6 +143,16 @@ def cluster_dict(self):
         for i in range(self.num_clusters):
             a = {'sections' : self.clusters[i]}
             self.tour_dict[i] = a
+        for i in range(self.num_clusters):
+            start_places = []
+            end_places = []
+            for t in self.tour_dict[i]["sections"]:

```

Comment:
I'm confused here. The trips have start and end places, not sections. Sections have start and end stops. Or is that just legacy that the dict is called "sections"?